src/utils/lru-cache.js

```diff
@@ -69,13 +69,15 @@ export class LruCache {
   /**
    * @param {number|string} key
    * @param {T} payload The payload to cache.
+   * @return {T} Returns payload back
    */
   put(key, payload) {
     if (!this.has(key)) {
       this.size_++;
     }
     this.cache_[key] = {payload, access: this.access_};
     this.evict_();
+    return payload;
```

Comment:
Intentional change since this PR is no longer using the LRU cache?

Reply:
Yeah, kept it for convenience and consistency.